src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractServiceImpl.java

```diff
@@ -160,7 +162,7 @@ private PdpFeedFileBaseEntry buildPdpFeedFileBaseEntry(ConcurStandardAccountingE
         int totalReimbursementLineCount = 0;
         KualiDecimal totalReimbursementDollarAmount = KualiDecimal.ZERO;
         for (ConcurStandardAccountingExtractDetailLine line : concurStandardAccountingExtractFile.getConcurStandardAccountingExtractDetailLines()) {
-            if (StringUtils.equalsIgnoreCase(line.getPaymentCode(), ConcurConstants.PAYMENT_CODE_CASH)) {
+            if (shouldProcessSAELineToPDP(line)) {
                 totalReimbursementLineCount++;
                 totalReimbursementDollarAmount = totalReimbursementDollarAmount.add(line.getJournalAmount());
```

Comment:
Considering how my "clean" function works, I don't think this total is needed any longer.  That and I don't think this was named as well as it could have been causing some confusion.  Either way, I will clean this up.